scout/load/all.py

```diff
@@ -53,85 +54,32 @@ def load_region(adapter, case_id, hgnc_id=None, chrom=None, start=None, end=None
         start = gene_caption["start"]
         end = gene_caption["end"]
 
-    LOG.info(
```

Comment:
refactored - so many repetitions, and we maintain the FILE_TYPE_MAP anyway.

Reply:
Nice!